client/my-sites/site-settings/settings-performance/main.jsx

```diff
@@ -128,6 +130,14 @@ class SiteSettingsPerformance extends Component {
 						fields={ fields }
 					/>
 				) }
+
+				{ ( ! siteIsJetpack || siteIsAtomic ) && (
```

Comment:
Is it possible that a site has `siteIsJetpack` and `siteIsAtomic` both set to true?

Reply:
Yes, naturally Atomic sites return `true` with `siteIsJetpack`. The goal of this check is to apply this to Simple and Atomic sites only. If I did just `! siteIsJetpack` then I get it on Simple site but not Atomic. But maybe I am missing something?

Reply:
No you're not missing anything, that's just me that wanted to check this out of curiosity. Good to go!